asapdiscovery-data/asapdiscovery/data/tests/test-pdb-download.py

```diff
@@ -1,10 +1,14 @@
+import sys
+
+sys.path.append(
+    "/Users/alexpayne/Scientific_Projects/covid-moonshot-ml/asapdiscovery-data"
+)
```

Comment:
Adding the path to the sys shouldn't be needed. Doing `pip install -e .` on `asapdiscover-data` should be able to handle things. We also don't want hardcoded paths from specific users.

Reply:
deleted in current version!

Reply:
Deleted in the current version!